CMS/templates/partials/resource_list_item.html

```diff
@@ -16,8 +16,7 @@
                 <div class="resource-card__title-container">
                     <h3 class="resource-card__title"
                         title="{{ resource_page.title }}">{{ resource_page.title }}</h3>
-                    <p class="resource-card__detail" title="Coronavirus (COVID-19)">Coronavirus
-                        (COVID-19)</p>
+                    <p class="resource-card__detail" title="Coronavirus (COVID-19)">{{ page.site_heading }}</p>
```

Comment:
The page variable here refers to the AssetTypePage not the individual resources.
These needs to be resource_page instead (and I don't think the function exists on the ResourceItemPage so will need to be added)